javascript/ql/src/semmle/javascript/frameworks/SQL.qll

```diff
@@ -31,11 +31,17 @@ private module MySql {
   /** Gets the package name `mysql` or `mysql2`. */
   API::Node mysql() { result = API::moduleImport(["mysql", "mysql2"]) }
 
+  /** Gets a reference to `mysql.createConnection`. */
+  API::Node createConnectionCallee() { result = mysql().getMember("createConnection") }
+
   /** Gets a call to `mysql.createConnection`. */
-  API::Node createConnection() { result = mysql().getMember("createConnection").getReturn() }
+  API::Node createConnection() { result = createConnectionCallee().getReturn() }
+
+  /** Gets a reference to `mysql.createPool`. */
+  API::Node createPoolCallee() { result = mysql().getMember("createPool") }
 
   /** Gets a call to `mysql.createPool`. */
```

Comment:
In addition to renaming the predicate, I'd suggest also updating the doc comment to more clearly emphasise that this doesn't just syntactically match calls to `createPool` but also does type tracking.